engine/table/src/main/java/io/deephaven/engine/table/impl/by/FormulaChunkedOperator.java

```diff
@@ -69,10 +71,13 @@ class FormulaChunkedOperator implements IterativeChunkedAggregationOperator {
      * @param columnParamName The token to substitute column names for
      * @param resultColumnPairs The names for formula input and result columns
      */
-    FormulaChunkedOperator(@NotNull final GroupByChunkedOperator groupBy,
+    FormulaChunkedOperator(
+            @NotNull final GroupByChunkedOperator groupBy,
             final boolean delegateToBy,
             @NotNull final String formula,
             @NotNull final String columnParamName,
+            @NotNull final Supplier<Map<String, Object>> queryScopeVariables,
+            @NotNull final QueryCompilerRequestProcessor compilationProcessor,
```

Comment:
Would it be cool to combine these into one object, so you don't always have to pass both?